### PR TITLE
Add 'Send me a text' button

### PR DESCRIPTION
**Pull request by chloepouprom**
Clicking on the 'Send me a text' button (only visible when on computer) will make a call to `/appUsers/:appUserId/integrations/twilio/ping`

I also added some more error handling.

@jugarrit @dannytranlx @alavers @lemieux

### CODE REVIEW

File: src/js/services/integrations-service.js [hunk 1]
```diff
@@ -63,10 +63,17 @@ export function linkTwilioChannel(userId, data) {
...
             });
         })
-        .catch(() => {
+        .catch((e) => {
+            const {response: {status}} = e;
+            let errorMessage;
+            if (status === 429) {
+                errorMessage = 'A link was attempted recently for this phone number. Please wait before trying again.';
+            } else {
+                errorMessage = 'We were unable to communicate with this number. Please enter a different one.';
```
Comment (jugarrit) [inline src/js/services/integrations-service.js]: If a `400` is returned this is the appropriate error, but if it's `500` maybe a try again might be good

File: src/js/components/channels/twilio-channel-content.jsx [hunk 2]
```diff
@@ -97,10 +93,10 @@ export class TwilioChannelContentComponent extends Component {
                                  </div>;
 
         const sendTextUrl = `sms://${phoneNumber}`;
-        const linkedComponentButton = isMobile.phone ? <button className='btn btn-sk-primary twilio-linking'
-                                                               onClick={ this.onStartTexting }>
-                                                           Start Texting
-                                                       </button> :
+        const linkedComponentButton = isMobile.phone ? <a href={ sendTextUrl }>
```
Comment (lemieux) [inline src/js/components/channels/twilio-channel-content.jsx]: why is there a button inside of a `a`?

**Comment by lemieux** [comment]
other than the button thing :+1:

**Comment by mspensieri** [comment]
All these new strings should live in the string constants dictionary, and should be customizable using the `customText` init property

**Comment by mspensieri** [comment]
👍

**Comment by jugarrit** [comment]
:+1:

**Pull request merged**